pootle/settings/90-dev-local.conf.sample

```diff
@@ -248,3 +248,6 @@ MT_BACKENDS = [
 
 # URL for the amaGama TM server.
 AMAGAMA_URL = 'https://amagama-live.translatehouse.org/api/v1/'
+
+# Silence the DEBUG check on dev servers
+SILENCED_SYSTEM_CHECKS = ["pootle.W005"]
```

Comment:
After discussion on IRC, we decided to leave it like this and consider changing it or removing it if people complain about it.